ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -846,6 +982,34 @@ const NetworksForm = ({
       }),
     );
   };
+
+  const isPopularNetwork = Object.values(FEATURED_RPCS).some(
+    (network) =>
+      getDisplayChainId(chainId) ===
+        parseInt(network.chainId, 16).toString(10) && rpcUrl === network.rpcUrl,
```

Comment:
I see many uses of `parseInt(..., 16).toString(10)` in this file; can we create a helper function with a meaningful name to make it more apparent what's happening here?